.github/workflows/tests.yml

```diff
@@ -139,6 +139,12 @@ jobs:
           pnpm install
           echo "::endgroup::"
 
+      - name: Packages install
+        run: |
+          echo "::group::Packages"
+          pnpx jetpack install --all -v --exclude packages/analyzer
```

Comment:
Ugh. As I said in Slack, IMO it's better to not bother to try to install deps for projects that aren't going to be tested.

Even worse is then maintaining a list of `--exclude` here to exclude certain ones that don't work.

Reply:
I agree it's not great, but it's also not great that a lot of our packages are skipped because of magical reasons which is why we didn't see this error before. And of course per the patch it's also not great that we run a super heavyweight mostly-unnecessary process every time you run `composer run phpunit`.

I removed this arg in favour of doing an explicit `composer update` during the test run in the github action. This seems cleaner anyway. Thanks for the feedback!